tests/cli/test_main_remove.py

```diff
@@ -114,16 +114,13 @@ def test_remove_nonexistent_env(conda_cli: CondaCLIFixture):
 def test_remove_all_default_activation_env(
     conda_cli: CondaCLIFixture,
     tmp_env: TmpEnvFixture,
-    tmp_path: Path,
+    monkeypatch: MonkeyPatch,
 ):
     """Check that removing the default_activation_env raises an exception."""
     with tmp_env() as prefix:
-        conda_cli(
-            "config",
-            "--set",
-            "default_activation_env",
-            prefix,
-        )
+        monkeypatch.setenv("CONDA_DEFAULT_ACTIVATION_ENV", prefix)
```

Comment:
I think this approach of setting config for tests is more consistent with rest of the test suite